src/parser/jobs/ast/modules/Sect/Sect.tsx

```diff
@@ -142,8 +142,9 @@ export default class Sect extends Module {
 	}
 
 	private onCast(event: CastEvent) {
-		// If they casted a sect at anytime it means they pulled without one on
-		this.pullWithoutSect = true
+		// If they used a sect after the fight started, it means they pulled without one on; otherwise, it was likely added by the PrecastStatus module.
+		this.pullWithoutSect = event.timestamp >= this.parser.eventTimeOffset
```

Comment:
Any reason to use `eventTimeOffset` over `start_time`?

Reply:
Nope

> Question on your timestamp aside, the changelog message could really be clearer as to what it's doing. I'm not sure how Precast could cause the issue you're talking about in the comment though unless they remove it after combat, because Precast relies ons seeing the status fall off.

I thought the aspected heals had a `statusesApplied` section, but looking at it they actually don't. It's being done by this module's own `normalize`.